homeassistant/components/neato/__init__.py

```diff
@@ -37,16 +38,16 @@
 }
 
 ACTION = {
-    0: 'Invalid',
-    1: 'House Cleaning',
-    2: 'Spot Cleaning',
-    3: 'Manual Cleaning',
+    0: 'No action',
```

Comment:
The changes to `ACTION` is a breaking change. Why should we make them?

Reply:
Done, I reverted